src/batches/entities/batch.entity.ts

```diff
@@ -0,0 +1,38 @@
+import { Batch, Prisma } from '@prisma/client';
```

Comment:
What is this `Batch` import?

Reply:
It is basically the Model Batch defined as the following (can be modified as per the finalized schema):

```
export type Batch = {
  id: string
  createdAt: Date
  updatedAt: Date
  name: string
  description: string | null
  templateType: string
  templateInput: string
  dataInput: string
  status: string
  output: string
}
```

Reply:
Okay, I mean where is it defined?